### ISSUE TITLE
Feature and home screen 

### ISSUE THREAD

**Issue by hunterscop1** [ItsMadruga/MdgWa#77]
Why my search bar missing and disable pinned massage limit not working in latest debug version?

**Comment by cvnertnc** [comment]
please update mgdwa debug app

**Comment by hunterscop1** [comment]
> please update mgdwa debug app

The search button is fixed but the disable pinned limit still not working

**Comment by cvnertnc** [comment]
> > please update mgdwa debug app
> 
> The search button is fixed but the disable pinned limit still not working

Restart whatsapp when you activate or deactivate it. To restart whatsapp, use the restart setting in the upper right three dots when whatsapp is on the main screen. I use it this way and there is no bug.

**Comment by hunterscop1** [comment]
> > > please update mgdwa debug app
> > 
> > The search button is fixed but the disable pinned limit still not working
> 
> Restart whatsapp when you activate or deactivate it. To restart whatsapp, use the restart setting in the upper right three dots when whatsapp is on the main screen. I use it this way and there is no bug.

Still not working

**Comment by cvnertnc** [comment]
> > > > please update mgdwa debug app
> > > 
> > > 
> > > The search button is fixed but the disable pinned limit still not working
> > 
> > 
> > Restart whatsapp when you activate or deactivate it. To restart whatsapp, use the restart setting in the upper right three dots when whatsapp is on the main screen. I use it this way and there is no bug.
> 
> Still not working

I'm using the same version too. No, I plan to release an update soon It's like this until now. Which WhatsApp version are you using? I am 2.24.8.85 use

**Comment by hunterscop1** [comment]
> > > > > please update mgdwa debug app
> > > > 
> > > > 
> > > > The search button is fixed but the disable pinned limit still not working
> > > 
> > > 
> > > Restart whatsapp when you activate or deactivate it. To restart whatsapp, use the restart setting in the upper right three dots when whatsapp is on the main screen. I use it this way and there is no bug.
> > 
> > Still not working
> 
> I'm using the same version too. No, I plan to release an update soon It's like this until now. Which WhatsApp version are you using? I am 2.24.8.85 use

Yes, I use the same version of Whatsapp that's supported on module
